iosApp/Survey/Sources/Presentation/Coordinator/RouteCoordinator.swift

```diff
@@ -47,3 +47,5 @@ extension RouteCoordinator: LoginCoordinator {
         routes = [.root(.surveySelection)]
     }
 }
+
+extension RouteCoordinator: AccountCoordinator {}
```

Comment:
There is a `showLogin` function for `AccountCoordinator` but it looks like it is already defined in `extension RouteCoordinator: SplashCoordinator` and it is okay to use the same implementation, right? Then should we combine `SplashCoordinator` and `AccountCoordinator` into the same extension and cleanup this new empty extension instead? What do you think? 🤔

Reply:
The idea is that each view should have respective Coordinator, if we use `SplashCoordinator` for `AccountCoordinator` and we want to add new feature to `AccountCoordinator` only SplashCoordinator will be affected.

Reply:
@blyscuit then maybe the function should be separated somehow (can using different names here), as defining an empty extension while there is an interface function defined in `AccountCoordinator` protocol creates a weird confusion, especially when this file gets bigger and harder to keep track imo e.g. which function belongs to which coordinator.